src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSectionRootRenderer.cs

```diff
@@ -130,7 +130,21 @@ public override void ViewSafeAreaInsetsDidChange()
 			LayoutHeader();
 		}
 
+		public override void TraitCollectionDidChange(UITraitCollection previousTraitCollection)
+		{
+			base.TraitCollectionDidChange(previousTraitCollection);
+
+			var currentItem = ShellSection.CurrentItem;
+
+			if (!_renderers.ContainsKey(currentItem))
+				return;
+
+			var currentItemHandler = _renderers[currentItem];
```

Comment:
We cna do a little better than this. If you noticed (and I believe this is a dict) you are doing  a look up twice. You can reduce that with the TryGetValue method:
```suggestion
			if (!_renderers.TryGetValue(currentItem, out var currentItemHandler))
				return;
```
You'll get the same out var and you'll remove the extra lookup.